sdk/cosmos/azure_data_cosmos/src/pipeline/authorization_policy.rs

```diff
@@ -21,7 +21,7 @@ use url::form_urlencoded;
 #[cfg(feature = "key_auth")]
 use azure_core::{credentials::Secret, hmac::hmac_sha256};
 
-const AZURE_VERSION: &str = "2018-12-31";
+const AZURE_VERSION: &str = "2020-07-15";
```

Comment:
Creating containers with hierarchical partition keys requires this version. I grabbed the string from the JS SDK.